python/instrumentation/openinference-instrumentation-mistralai/tests/openinference/instrumentation/mistralai/test_instrumentor.py

```diff
@@ -818,21 +843,18 @@ def test_synchronous_streaming_chat_completions_emits_expected_span(
     prompt_template_version: str,
     prompt_template_variables: Dict[str, Any],
 ) -> None:
-    respx.post("https://api.mistral.ai/v1/chat/completions").mock(
-        return_value=Response(
-            200,
-            stream=chat_stream,
-        )
-    )
-
-    def mistral_stream() -> Iterable[ChatCompletionStreamResponse]:
-        return mistral_sync_client.chat_stream(
-            model="mistral-large-latest",
-            messages=[
-                ChatMessage(
-                    content="Who won the World Cup in 2018? Answer in one word, no punctuation.",
-                    role="user",
-                )
+    def mistral_stream() -> Generator[CompletionEvent, None, None]:
+        mistral_client = Mistral(api_key="redacted")
+        return mistral_client.chat.stream(  # type: ignore
```

Comment:
can we assert that the instrumented and uninstrumented stream both pass a type check a user might expect? (both for sync and async tests)